pkg/apiserver/apiserver.go

```diff
@@ -28,7 +28,7 @@ import (
 	configv1alpha2 "kubesphere.io/kubesphere/pkg/kapis/config/v1alpha2"
 	iamv1alpha2 "kubesphere.io/kubesphere/pkg/kapis/iam/v1alpha2"
 	loggingv1alpha2 "kubesphere.io/kubesphere/pkg/kapis/logging/v1alpha2"
-	monitoringv1alpha2 "kubesphere.io/kubesphere/pkg/kapis/monitoring/v1alpha2"
+	monitoringv1alpha2 "kubesphere.io/kubesphere/pkg/kapis/monitoring/v1alpha3"
```

Comment:
monitoringv1alpha2 doesn't match "kubesphere.io/kubesphere/pkg/kapis/monitoring/v1alpha3"
if it's indeed v1alpha3, maybe we can change it to something like v1beta1 ?

Reply:
As the monitoring api group is growing, I think we are still at the alpha stage. For example new apis for custom monitoring will come in soon.

Reply:
done